segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -105,25 +109,27 @@
      * @param store      The SegmentStore to send append requests to.
      * @param connection The ServerConnection to send responses to.
      * @param next       The RequestProcessor to invoke next.
+     * @param verifier    The token verifier.
      */
     @VisibleForTesting
-    public AppendProcessor(StreamSegmentStore store, ServerConnection connection, RequestProcessor next) {
-        this(store, connection, next, null);
+    public AppendProcessor(StreamSegmentStore store, ServerConnection connection, RequestProcessor next, DelegationTokenVerifier verifier) {
+        this(store, connection, next, null, verifier);
     }
 
     /**
      * Creates a new instance of the AppendProcessor class.
-     *
-     * @param store         The SegmentStore to send append requests to.
+     *  @param store         The SegmentStore to send append requests to.
      * @param connection    The ServerConnection to send responses to.
      * @param next          The RequestProcessor to invoke next.
      * @param statsRecorder (Optional) A StatsRecorder to record Metrics.
+     * @param tokenVerifier Delegation token verifier.
      */
-    AppendProcessor(StreamSegmentStore store, ServerConnection connection, RequestProcessor next, SegmentStatsRecorder statsRecorder) {
+    AppendProcessor(StreamSegmentStore store, ServerConnection connection, RequestProcessor next, SegmentStatsRecorder statsRecorder, DelegationTokenVerifier tokenVerifier) {
         this.store = Preconditions.checkNotNull(store, "store");
         this.connection = Preconditions.checkNotNull(connection, "connection");
         this.nextRequestProcessor = Preconditions.checkNotNull(next, "next");
         this.statsRecorder = statsRecorder;
+        this.tokenVerifier = tokenVerifier;
```

Comment:
I see this can be null (from the impl below). If null, let's replace it with a private no-op implementation so we don't have to check for null every time we use it.

Reply:
There is only one place where this is used.